drbd.mysql-example.bom

```diff
@@ -0,0 +1,113 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  version: "1.0.0-SNAPSHOT" # BROOKLYN_DRBD_VERSION
+  description: |
+    This is an example DRBD database application. MySQL instances are mirrored on DRBD with Jetty runners pointing at the primary
+  publish:
+    license_code: Apache-2.0
+  items: 
+  - id: example-drbd-mysql
+    item: 
+      type: org.apache.brooklyn.entity.database.mysql.MySqlNode
+      brooklyn.config:
+        mysql.password: s0m3rand0mp4ss
+        latch.install: $brooklyn:parent().parent().attributeWhenReady("service.isUp")
```

Comment:
Feels like this should happen by default?

Reply:
Not nessacarily because the children are detached and DRBD restarts the machine